### ISSUE TITLE
How to prevent php json patch convert chinese to unicode string?

### ISSUE THREAD

**Issue by netroby** [raphaelstolt/php-jsonpatch#21]
I am try to using php jsonpatch, it works fine , but it will force convert chinese words to unicode, this will lead problems in our system . how to prevent it force convert encoding?

```
Old json
string(37) "{"bookid":"1","bookname":"第二个"}"
Patch json
string(108) "[{"op":"replace", "path":"/bookid", "value":"0"}, {"op":"replace", "path":"/bookname", "value":"第一个"}]"
Patched json
string(46) "{"bookid":"0","bookname":"\u7b2c\u4e00\u4e2a"}"
```

**Comment by raphaelstolt** [comment]
Hi @netroby, this could be fixed by raising the required PHP version to 5.4 which allows to pass the `JSON_UNESCAPED_UNICODE` option to `json_encode`, which fixes your described problem. 

Which PHP version are you using?

**Comment by netroby** [comment]
PHP 7.0.6
